src/rime/gear/key_binder.cc

```diff
@@ -203,7 +212,9 @@ void KeyBinder::PerformKeyBinding(const KeyBinding& binding) {
   }
   else {
     redirecting_ = true;
-    engine_->ProcessKey(binding.target);
+    for (const KeyEvent& key_event: binding.target) {
```

Comment:
nit:
we follow [Google C++ Style](https://google.github.io/styleguide/cppguide.html#Horizontal_Whitespace), which has the following style preference for Loops and Conditionals
```c++
// Range-based for loops always have a space before and after the colon.
for (auto x : counts) {
  ...
}
```